src/main/java/com/google/api/gax/core/AsyncPage.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2016, Google Inc. All rights reserved.
```

Comment:
Done. The abstract classes I think are hidding renames of the e.g. PageImpl classes